crates/codegen/src/csharp.rs

```diff
@@ -355,7 +355,7 @@ const REDUCER_EVENTS: &str = r#"
         /// <summary>
         /// The procedure event that caused this callback to run.
         /// </summary>
-        public readonly ProcedureEvent Event;
+        public ProcedureEvent Event { get; }
```

Comment:
Why do we have changes in this file? I'd expect this PR not to affect client codegen, or anything about the client SDK, at all.